### PR TITLE
Qt state

### PR DESCRIPTION
**Pull request by Zer0xFF**
- [x] Windows
- [x] MacOS
- [x] Linux (requires https://github.com/jpd002/Play--Framework/pull/33)

this seems to be working well on Windows/MacOS and is also working on Linux (partially requires PR).

Just to note, this https://github.com/jpd002/Play-/pull/1168/files#diff-339f49333be859a94e15ae84bfb784283133b333f468281a3003b2e9f8d3cdb8R162 can be "improved" performance wise if we instead left join in the 2 functions that `select * from bootables`, but considering the size of the db, the speed diff is almost going to be negliable, i opted for cleaner code instead.

### CODE REVIEW

File: .github/CI_MISC/build_compatibility_db.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+def updateDatabase(con, games, labels):
+	con.execute('BEGIN')
+	cur = con.cursor()
+
+	cur.execute("DELETE FROM labels;")
+	for label in labels:
+			cur.execute("INSERT OR IGNORE INTO labels(name, color) values(?, ?);", (label['name'], label['color'],))
+
+	for game in games:
+		print(f"Updating {game['discId']}")
```
Comment (jpd002) [inline .github/CI_MISC/build_compatibility_db.py:94]: I would remove printing every single disc id on the console. There's 1.5k right now and it's just going to grow.

File: .github/CI_MISC/build_compatibility_db.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+	URL = "https://api.github.com/repos/jpd002/Play-compatibility/issues?state=open&per_page=100&since={}&page={}"
+
+	today = datetime.date.today()
+	yesterday = today - datetime.timedelta(days=2)
+
+	i = 1
+	res = []
+	hasNext = True
+	while(hasNext):
+		print(f"Retrieving Page {i}")
```
Comment (jpd002) [inline .github/CI_MISC/build_compatibility_db.py:57]: I would remove this as it doesn't really give interesting information (like for disc ids).

**Comment by jpd002** [comment]
Added a few comments, but other than that, this is looking good!

**Comment by jpd002** [comment]
Thanks a lot for this!

**Comment by Zer0xFF** [comment]
my pleasure :)

**Pull request merged**